scss/standalone/patterns_media-container.scss

```
@@ -0,0 +1,5 @@
+@import '../vanilla';
+@include vf-base;
+
+@include vf-b-hr;
```

Including `vf-b-hr` is not needed (it's already part of `vf-base`), but what we need is the rule component styles:

```suggestion
@include vf-p-rule;
```